client/src/redux/reducers/workbench.reducers.js

```diff
@@ -1,13 +1,21 @@
 export const initialState = {
-  debug: false,
+  loading: false,
+  config: {},
```

Comment:
Hypothetical, rather than a required change. For "enabled" options (such as view path, create link, copy to public, etc.) would we be explicitly putting in those settings as true/false into the reducer state or just allowing them to be undefined/falsy? 

In other words, are you expecting us to do be explicit or dynamic with these settings?

Reply:
That's a great q, and one I'd like input on. My thoughts were keep this field dynamic, and if a value is falsy, the component has its answer

Reply:
I prefer falsy, since it could prevent some misconfiguration from affecting the client.

Reply:
By that do you mean you prefer this solution as is?

Reply:
Yep